server/index.js

```diff
@@ -0,0 +1,205 @@
+require('dotenv').config();
+const request = require('request');
+const http = require('http');
+const httpProxy = require('http-proxy');
+const express = require('express');
+const bodyParser = require('body-parser');
+const bcrypt = require('bcrypt');
+const JWT = require('./jwt');
+
+const app = express();
+
+const proxy = httpProxy.createProxyServer({
+  changeOrigin: true,
+  ignorePath: false
+});
+
+proxy.on('proxyReq', function(proxyReq, req, res, options) {
+  proxyReq.setHeader('authorization', `Bearer ${process.env.AIRTABLE_API_KEY}`);
+});
+
+app.use(bodyParser.json());
+
+app.get('/v0*', JWT.verifyToken, (req, res) => {
+  proxy.web(req, res, {
+    target: `${process.env.AIRTABLE_ENDPOINT_URL}`
+  });
+});
+
+app.post('/v0*', JWT.verifyToken, (req, res) => {
+  proxy.web(req, res, {
+    target: `${process.env.AIRTABLE_ENDPOINT_URL}`
+  });
+});
+
+app.put('/v0*', JWT.verifyToken, (req, res) => {
+  proxy.web(req, res, {
+    target: `${process.env.AIRTABLE_ENDPOINT_URL}`
+  });
+});
+
+app.delete('/v0*', JWT.verifyToken, (req, res) => {
+  proxy.web(req, res, {
+    target: `${process.env.AIRTABLE_ENDPOINT_URL}`
+  });
+});
```

Comment:
Should these (app.get/post/put/delete) be turned into `app.use`?

Reply:
Good call, I put the middleware in a separate line with `app.use` and put the proxy.web under `app.all('/v0*')`

I had to use app.all vs app.use because app.all will match all of the HTTP verbs and will allow express to preserve the /path which is passed on the the proxy server.